tests/std/tests/P2517R1_apply_conditional_noexcept/test.compile.pass.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <string>
+#include <tuple>
+#include <type_traits>
+
+using namespace std;
+
+struct NoexceptIf {
+    template <bool b, class... Ts>

```

Comment:
Style nitpick: We conventionally capitalize template parameters, so this should say `B` :bee:. (Test code is less strict about conventions, but this is still followed universally AFAIK.) This makes it easier to see at a glance which are template parameters vs. function parameters etc.